### PR TITLE
Add gatsby

### PR DESCRIPTION
**Pull request by mikkelcodes**

### CODE REVIEW

File: packages/shared-components/navigation/index.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import Link from 'gatsby-link';
+import * as Icon from '../icon';
+import * as Logo from '../logo';
+import { LargeLink, Subtitle, ListLink, List } from '../navItems';
+import { SocialLinks } from '../socialShare';
+import useWindowDimensions from '../../shared-hooks/useWindowDimensions';
```
Comment (TheKnarf) [inline packages/shared-components/navigation/index.js:7]: Is this import correct`

File: packages/shared-components/navigation/index.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
+import React from 'react';
```
Comment (TheKnarf) [inline packages/shared-components/navigation/index.js:1]: Why is this file outside of `src/`

File: packages/website-alvb/.storybook/hire/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```
Comment (TheKnarf) [inline packages/website-alvb/.storybook/hire/index.js:1]: Why is this file inside a `.storybook/` folder? Usually thats where you would find config files for storybook.

File: packages/website-alvb/gatsby-config.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
+require('dotenv').config({
+  path: `.env.${process.env.NODE_ENV}`,
+});
+
+module.exports = {
+  siteMetadata: {
+    siteUrl: `https://www.alv.no`,
```
Comment (TheKnarf) [inline packages/website-alvb/gatsby-config.js:7]: This url might be wrong for alvb.

File: packages/website-alvb/gatsby-config.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    title: `Alv Konsulentselskap`,
```
Comment (TheKnarf) [inline packages/website-alvb/gatsby-config.js:8]: Maybe update title and description as well?

File: packages/website-alvb/gatsby-node.js [hunk 6]
```diff
@@ -0,0 +1,240 @@
+// const path = require(`path`);
```
Comment (TheKnarf) [inline packages/website-alvb/gatsby-node.js:1]: Just remove this whole file from alv-b instead of leaving it outcommented.

**Comment by TheKnarf** [comment]
Is there any way that we can split this into two PR's? One that only moves things into `shared-components` and updates the main site. And one that adds the new site?

That way it might be easier to review and test that the old site still works.

**Comment by mikkelcodes** [comment]
It's possible to run both sites locally and verify that the functionality of the og site is the same as before

**Comment by mikkelcodes** [comment]
I can try to split it up

**Pull request closed without merging**